src/index.html

```diff
@@ -1,13 +1,260 @@
+              src="./images/product-bike3.svg"
+              alt="product bike #3"
+            >
+            <h2 class="main__secondary-page-title">Agile ride 3</h2>
+            <p class="main__paragraph">
+              The lightweight frame that has earned its
+              street tread as a sleek, agile ride.
+            </p>
+            <h3 class="compare-bikes__price">$ 2 090</h3>
+          </article>
+        </div>
+      </section>
+
+      <section class="details" id="details">
+        <h1 class="main__page-title">The Details</h1>
+
+        <div class="details__content">
+          <article class="detail">
+            <div class="detail__images">
+              <a class="detail__link detail__link--1" href="#">
+                <img
+                  class="detail__photo"
+                  src="./images/details-1.svg"
+                  alt="bike details 1"
+                >
+              </a>
+              <a class="detail__link detail__link--2" href="#">
+                <img
+                  class="detail__photo"
+                  src="./images/details-2.svg"
+                  alt="bike details 2"
+                >
+              </a>
+            </div>
+
+            <div class="detail__description">
+              <h2 class="main__secondary-page-title">Auto Unlock</h2>
+              <p class="main__paragraph">
+                The app senses when you're nearby to unlock automatically.
+                GPS tracking so you know where your bike is and can track it anytime.
+              </p>
+            </div>
+          </article>
+
+          <article class="detail">
+            <div class="detail__images">
+              <a class="detail__link detail__link--1" href="#">
+                <img
+                  class="detail__photo"
+                  src="./images/details-3.svg"
+                  alt="bike details 3"
+                >
+              </a>
+              <a class="detail__link detail__link--2" href="#">
+                <img
+                  class="detail__photo"
+                  src="./images/details-4.svg"
+                  alt="bike details 4"
+                >
+              </a>
+            </div>
+
+            <div class="detail__description">
+              <h2 class="main__secondary-page-title">Range & Integrated lights</h2>
+              <p class="main__paragraph">
+                The removable battery has up to 70km battery autonomy
+                and weighs only 2.4 kg. Lights integrated into the frame
+                give you always-on visibility day and night.
+              </p>
+            </div>
+          </article>
+
+          <article class="detail">
+            <div class="detail__images">
+              <a class="detail__link detail__link--2" href="#">
+                <img
+                class="detail__photo"
+                  src="./images/details-5.svg"
+                  alt="bike details 5"
+                >
```

Comment:
check formatting please
```suggestion
                <img
                  class="detail__photo"
                  src="./images/details-5.svg"
                  alt="bike details 5"
                >
```